browser/brave_shields/ad_block_service_browsertest.cc

```diff
@@ -3039,20 +3043,22 @@ IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, ContentPicker) {
 
   // Reload the page and check the selector is blocked by the new rule.
   NavigateToURL(tab_url);
-  WaitForSelectorBlocked(web_contents(), "#ad-banner");
+  WaitForSelector(web_contents(), "#ad-banner");
```

Comment:
I'd prefer to avoid `WaitForSelectorBlocked` to `WaitForSelector` changes, now it's not immediatly clear what we're waiting for. I think it would be better to reimplement `WaitForSelectorBlocked` via WaitForSelector. May be we should also implement `WaitForSelectorVisible` as well